### PR TITLE
Update Vite 5

### PR DESCRIPTION
**Pull request by Sakchai-Refinitiv**
## Description

Update Vite 5 improve performance and security.

Fixes # (issue)
[ELF-2354](https://jira.refinitiv.com/browse/ELF-2354)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,8 +49,8 @@
     "nx": "^16.9.1",
     "sinon": "^15.1.0",
     "typescript": "^4.8.3",
-    "vite": "^4.4.9",
-    "web-component-analyzer": "2.0.0-next.4",
+    "vite": "^5.0.10",
+    "web-component-analyzer": "2.0.0",
```
Comment (wsuwt) [inline package.json:53]: Is there any reason to this version update ? Please check with Mos, this may potentially introduce bug in API generation.

Comment (Sakchai-Refinitiv) [reply package.json:53]: It has a security fix in the package, I think it should compatible with 2.0.0. @Sarin-Udompanish  Can you help me check it?

Comment (Sarin-Udompanish) [reply package.json:53]: LGTM
Screenshot 2566-12-21 at 16 35 28

**Review by wsuwt — APPROVED** [review]
Looks good to me.

**Pull request merged**